components/translation/usage.rst

```diff
@@ -21,79 +21,6 @@ In this example, the message *"Symfony is great!"* will be translated into
 the locale set in the constructor (``fr_FR``) if the message exists in one of
 the message catalogs.
 
-.. _component-translation-placeholders:
```

Comment:
shall we keep this anchor?

Reply:
In a strict way it is kept, but then in `/translation/message_format`. I don't think it's useful to keep them on this page (it's not documented anymore on this page).

Reply:
Yes I meant tinker pot on another place but in this case the url differs, so no need to keep the same anchor....